src/vs/workbench/contrib/debug/browser/loadedScriptsView.ts

```diff
@@ -764,3 +771,25 @@ class LoadedScriptsFilter implements ITreeFilter<BaseTreeItem, FuzzyScore> {
 		return TreeVisibility.Recurse;
 	}
 }
+registerAction2(class Collapse extends ViewAction<LoadedScriptsView> {
+	constructor() {
+		super({
+			id: 'loadedScripts.collapse',
+			viewId: LOADED_SCRIPTS_VIEW_ID,
+			title: nls.localize('collapse', "Collapse All"),
+			f1: false,
+			icon: Codicon.collapseAll,
+			precondition: CONTEXT_DEBUG_STATE.isEqualTo(getStateLabel(State.Stopped)),
```

Comment:
Why this precondition? I can use Loaded Scripts even when not paused.

Reply:
@roblourens Good call! This was indeed not needed, removed the condition in this commit: https://github.com/microsoft/vscode/pull/203560/commits/30262b96022f938121f98594a4657aab819e4053